src/celeritas/user/ActionDiagnostic.hh

```diff
@@ -101,6 +104,7 @@ class ActionDiagnostic final : public ExplicitActionInterface,
     using StoreT = StreamStore<ParticleTallyParamsData, ParticleTallyStateData>;
 
     ActionId id_;
+    bool initialized_{false};
```

Comment:
Is this redundant with `bool(store_)`?

Reply:
Actually yeah you're right, if the store is initialized that should guarantee we also have the particle params and action registry.